### ISSUE TITLE
Typescript

### ISSUE THREAD

**Issue by leungandrew** [Annoraaq/phaser-grid-movement-plugin#16]
Hi there!

Looking to try out your plugin into a Typescript project.  Would be great if you can expose type declarations along with your package!

```
Could not find a declaration file for module 'phaser-grid-movement-plugin'. '/myproject/node_modules/phaser-grid-movement-plugin/dist/GridMovementPlugin.min.js' implicitly has an 'any' type.
  Try `npm install @types/phaser-grid-movement-plugin` if it exists or add a new declaration (.d.ts) file containing `declare module 'phaser-grid-movement-plugin';`  TS7016
```

**Comment by Annoraaq** [comment]
Hi leungandrew, I thought I was already doing this. Let me check, whats the issue.

**Comment by Annoraaq** [comment]
I can reproduce the issue. Thank you for pointing it out 👍 I will fix it

**Comment by Annoraaq** [comment]
Should be fixed and published as version 1.2.2
